ddp-workspace/projects/ddp-pancan/src/app/components/app/app.component.ts

```diff
@@ -1,20 +1,38 @@
 import { Component, Inject } from '@angular/core';
-import { AppRedesignedBaseComponent, CommunicationService, ToolkitConfigurationService } from 'toolkit';
 import { MatDialog } from '@angular/material/dialog';
+import { NavigationEnd, Router } from '@angular/router';
+import {
+    AppRedesignedBaseComponent,
+    CommunicationService,
+    ToolkitConfigurationService
+} from 'toolkit';
 import { RenewSessionNotifier } from 'ddp-sdk';
-import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
 })
 export class AppComponent extends AppRedesignedBaseComponent {
+    private readonly splashPageUrls = ['colorectal'];
```

Comment:
what splash page means?

Reply:
Don't know what exactly the term `splash` means in this case.
However there are 2 pages [(Colorectal,](https://broadinstitute.atlassian.net/browse/DDP-6640) [Leiomyosarcoma)](https://broadinstitute.atlassian.net/browse/DDP-6667) that should have a separate landing page with a bit different from Pancan welcome page design.